apitests/spec/device_actions_spec.ts

```diff
@@ -0,0 +1,23 @@
+describe('typescript test', () => {
```

Comment:
I'll delete the tests inside that file and add correct ones (see TODO). Sending it like this so that we can all start using typescript, gulp,...